app/controllers/events_controller.rb

```diff
@@ -1,3 +1,7 @@
 class EventsController < ApplicationController
-  def index; end
+  #This method will show the events in /events view
+  def events
+    @events = Event.all
+  end
```

Comment:
Why did you delete the restful methods?

Reply:
The index method didn't do anything, instead the events method does the request of the data, and show it in events view.

Reply:
its better to do restful methods instead of that events its better to do index